.gitignore

```diff
@@ -7,6 +7,7 @@
 /pkg/
 /spec/reports/
 /tmp/
+/vendor
```

Comment:
I'd rather not add this. `bundler` should put the gems into `.bundle` or in a global cache. Or is this added for another reason?